### PR TITLE
doc: Update vstart section in readme.md

### PR DESCRIPTION
**Pull request by Dedsec0098**
doc: Gave more details about tips and tricks and troubleshooting for Vstsrt in the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,37 @@ To build Ceph, follow this procedure:
...
+    >
+    >Use Debug Builds Only When Needed 
+    >
+    >While debugging builds are helpful for development, they can slow down performance.
+    >
+    >Use `-DCMAKE_BUILD_TYPE=Release` when debugging isn't necessary.
+    >
+    >Enable Selective Daemons if you're testing specific components, don't start unnecessary daemons.
+    >
+    >Preserve Existing Data skip cluster reinitialization between tests by using the `-n` flag
```
Comment (zdover23) [inline README.md:141]: Line 141 should be rewritten as a sentence.

File: README.md [hunk 2]
```diff
@@ -126,6 +126,37 @@ To build Ceph, follow this procedure:
...
+
+    > [TROUBLESHOOTING]
+    >
+    >Cluster Fails to Start: Look for errors in the logs under the out/ directory, e.g.
+    >
+    >OSD Crashes: Check the OSD logs for errors.
+    >
+    >Cluster in a Health Error State: Run ceph status to identify the issue.
```
Comment (zdover23) [inline README.md:149]: "ceph status" should be in backticks.

**Review by zdover23 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zdover23) [reply README.md:141]: Line 139 should also be rewritten as a sentence.

**Comment by zdover23** [comment]
jenkins retest this please

**Comment by Dedsec0098** [comment]
Can you please elaborate on how can I fix this problem of test failing.

**Comment by zdover23** [comment]
> Can you please elaborate on how can I fix this problem of test failing.

Change the title of this PR to "doc: Update vstart section in readme.md". The "Signed-off-by" check expects the string "doc" at the beginning of the title. Use "git commit --amend" to achieve this.

I'll take a look at this tomorrow and fix it up if you haven't gotten to it by then.

**Comment by zdover23** [comment]
jenkins test make check

**Comment by zdover23** [comment]
jenkins test make check arm64

**Comment by zdover23** [comment]
jenkins test make check

**Comment by Dedsec0098** [comment]
I have done all the changes and also I have removed previous commits that were causing problems with the checks.

**Comment by zdover23** [comment]
jenkins retest this please

**Comment by zdover23** [comment]
> I have done all the changes and also I have removed previous commits that were causing problems with the checks.

This should be squashed down to a single commit with a title that begins with "doc:". I'll leave this to you for the next twelve hours, and then I'll make a go of it.

**Comment by zdover23** [comment]
jenkins test api

**Review by zdover23 — APPROVED** [review]
LGTM. I'll refine the English in a future PR.

**Pull request merged**